src/wiktextract/extractor/simple/section_titles.py

```diff
@@ -0,0 +1,114 @@
+            "irrnoun": ["irregular"],
+            "noun2": ["plural-only"],
+            "noun3": None,  # Two singular forms
+            "noun4": ["singular-only"],
+            "letter": ["letter"],
+        },
+    },
+    "abbreviation": {
+        "pos": "abbrev",
+        "tags": ["abbreviation"],
+        "templates": {"abbreviation": None},
+    },
+    "acronym": {
+        "pos": "abbrev",
+        "tags": ["abbreviation"],
+        "templates": {"acronym": None, "initialism": None},
+    },
+    "adjective": {"pos": "adj", "templates": {"adjective": None, "adj": None}},
+    "adverb": {"pos": "adv", "templates": {"adverb": None}},
+    "determiner": {
+        "pos": "det",
+        "templates": {
+            "comparative determiner": ["comparative"],
+            "determiner-comp": ["comparative"],
+            "determiner": None,
+        },
+    },
+    "conjunction": {"pos": "conj", "templates": {"conjunction": None}},
+    "contraction": {
+        "pos": "contraction",
+        "tags": ["abbreviation"],
+        "templates": {"contraction": None},
+    },
+    "coordinator": {
+        "pos": "conj",
+        "tags": ["coordinating"],
+        "templates": {"coordinator": None},
+    },
+    "expression": {"pos": "phrase", "templates": {"expression": None}},
+    "initialism": {
+        "pos": "abbrev",
+        "tags": ["abbreviation"],
+        "templates": {"initialism": None, "acronym": None},
+    },
+    "interjection": {"pos": "intj", "templates": {"interjection": None}},
+    "preposition": {"pos": "prep", "templates": {"preposition": None}},
+    "prefix": {
+        "pos": "prefix",
+        "tags": ["morpheme"],
+        "templates": {"prefix": None},
+    },
+    "pronoun": {"pos": "pron", "templates": {"pron": None}},
+    "proper noun": {"pos": "name", "templates": {"proper noun": None}},
+    "subordinator": {
+        "pos": "conj",
+        "tags": ["subordinate-clause"],
+        "templates": {"subordinator": None},
+    },
+    "suffix": {
+        "pos": "suffix",
+        "tags": ["morpheme"],
+        "templates": {"suffix": None},
+    },
+    "symbol": {"pos": "symbol", "templates": {"symbol": None}},
+    "verb": {
+        "pos": "verb",
+        "templates": {
+            "verb": None,
+            "verb2": None,  # alternative paradigms, "spelled" vs. "spelt"
+            "verb3": None,  # Only used by "be", "is", etc.
+            "verb4": ["auxiliary", "modal"],  # modal aux. 'will', 'can'.
+            "verb5": ["auxiliary"],  # non-modal auxiliares like 'do' and 'have'
+            "verb6": ["auxiliary"],  # same as verb5?
+        },
+    },
+}
+
+
+POS_HEADINGS_MAP = {
+    "acronym & initialism": "acronym",
+}
+
+# POS aliases; use this to figure out if a heading is a POS heading
+POS_HEADINGS = { k: v for k, v in POS_DATA.items() }
+
+for k, v in POS_HEADINGS_MAP.items():
+    if k in POS_HEADINGS or v not in POS_DATA:
+        continue
+    POS_HEADINGS[k] = POS_DATA[v]
```

Comment:
If only have one title needs to map, better add it directly to `POS_DATA`.

`POS_DATA["acronym & initialism"] = POS_DATA["acronym"]`

or

```python
for k, v in POS_HEADINGS_MAP.items():
    POS_DATA[k] = POS_DATA[v]
```

no need to duplicate whole dictionary for a few keys.

Reply:
Aaah, of course. I got stuck thinking about if it would be possible to do internal references like that when first writing out the dictionary literal, but somehow I never figured this.